terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/logging/TraceLoggingInterceptorTest.java

```diff
@@ -20,7 +20,9 @@
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
```

Comment:
- [ ] Is it a remnant of various considerations?
`org.mockito.Mockito.spy` seems unnecessary.

There may be other unnecessary items.

https://github.com/terasolunaorg/terasoluna-gfw/blob/fa558e768ee5f02479a7b14a6a0460c72baef1c3/terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/logging/TraceLoggingInterceptorTest.java#L67
https://github.com/terasolunaorg/terasoluna-gfw/blob/fa558e768ee5f02479a7b14a6a0460c72baef1c3/terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/logging/TraceLoggingInterceptorTest.java#L46
https://github.com/terasolunaorg/terasoluna-gfw/blob/fa558e768ee5f02479a7b14a6a0460c72baef1c3/terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/logging/TraceLoggingInterceptorTest.java#L31

Reply:
Delete unnecessary lines
37cb0a8